js/api.md

```diff
@@ -440,8 +440,8 @@ class App extends React.Component {
 
         return (
             <Connect query={graphqlOperation(ListEvents)}>
-                {({ data: { listEvents } }) => (
-                    <ListView events={listEvents.items} />
+                {({ data: { listEvents }, loading }) => (
```

Comment:
Can we do something a little more comprehensive that also includes error checking like so:

```
<Connect query={graphqlOperation(ListEvents)}>
  {({ data: { listEvents }, loading, error }) => (
    if (error) return <h1>Error fetching results</h1>
    if (loading || !data) return <h1>Loading...</h1>
      return <ListView events={listEvents.items} />
  )}
</Connect>
```

Reply:
For sure, I will try this and update the docs. Thanks!